sdks/java/extensions/google-cloud-platform-core/src/main/java/org/apache/beam/sdk/extensions/gcp/util/GcsUtil.java

```diff
@@ -454,8 +458,31 @@ public SeekableByteChannel open(GcsPath path) throws IOException {
   @VisibleForTesting
   SeekableByteChannel open(GcsPath path, GoogleCloudStorageReadOptions readOptions)
```

Comment:
@chamikaramj While randomly reading this code, I was wondering if this method is the one called in pipelines or the above version with 1 argument (line 445).